bindings/python/pydeck/pydeck/widget/widget.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+"""
+DeckGLWidget renders JSON into a deck.gl plot
+It requires a mapbox API key to be set as an environment variable
```

Comment:
Some of our layers are/can be used as non-geospatial. In such cases, it should be possible to use this library without a mapbox key.